README.md

```diff
@@ -71,7 +71,12 @@ require('yabs'):setup({
             -- WARNING: this api is experimental and subject to
             -- changes
             require('yabs'):run_task('build', {
-              on_exit = function(Job, exit_code) -- Job here is a plenary.job object
+              -- Job here is a plenary.job object that represents
+              -- the finished task, read more about it here:
+              -- https://github.com/nvim-lua/plenary.nvim/blob/master/lua/plenary/job.lua
```

Comment:
I think here we could also link to https://github.com/nvim-lua/plenary.nvim#plenaryjob.